packages/backend-deployer/API.md

```diff
@@ -41,6 +41,11 @@ export type DestroyResult = {
     deploymentTimes: DeploymentTimes;
 };
 
+// @public (undocumented)
+export type Formatter = {

```

Comment:
This type keeps the backend-deployer decoupled from any cli-specific packages

Reply:
Should this be called `BackendDeployerOutputsFormatter` or something more specific in that case?
Also should this be optional ?

Reply:
I'm open to different names but I think it should be required. Otherwise we just end up having forking logic later based on whether its defined or not.

However, another option could be to just rework [this error message](https://github.com/aws-amplify/amplify-backend/pull/1373/files#diff-986cd9a92cf0dda4ec2be12d91183a4a550e082c8b3b41846dbf2832a00d3c60R231) to point to docs instead of suggesting a command

Reply:
fwiw. we can make it optional in the future as minor version increment.

Reply:
but please rename.